CHANGELOG.md

```diff
@@ -26,7 +26,7 @@ BUG FIXES:
 - Syntax error in the `required_providers` block does not panic anymore, but yields "syntax error" ([2344](https://github.com/opentofu/opentofu/issues/2344))
 - Changing Go version to 1.22.11 in order to fix [CVE-2024-45336](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45336) and [CVE-2024-45341](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45341)
 - Fix the error message when default value of a complex variable is containing a wrong type ([2394](https://github.com/opentofu/opentofu/issues/2394))
-
+- Fixed the git referenced module with a branch that contains slashes in the name. ([2396](https://github.com/opentofu/opentofu/issues/2396))
```

Comment:
Let's update this to make it clear it only impacts the github addresses

Reply:
Applied. Thanks